pkg/apis/starrocks/v1alpha1/starrockscluster_types.go

```diff
@@ -40,6 +43,12 @@ type StarRocksClusterSpec struct {
 	StarRocksCnSpec *StarRocksCnSpec `json:"starRocksCnSpec,omitempty"`
 }
 
+//The start starrocks user info.
+type User struct {
+	Name               string                    `json:"name,omitempty"`
```

Comment:
use `UID`, hide PodSecurityContext from end user, operator will assemble the correct SecurityContext for PODs.